.github/workflows/setupapp.yml

```diff
@@ -148,6 +148,10 @@ jobs:
       run: |
         docker build -t localhost:5000/local_monai:latest -f Dockerfile .
         docker push localhost:5000/local_monai:latest
+        docker tag localhost:5000/local_monai:latest projectmonai/monai:latest
```

Comment:
is it possible to check that branch name == master before this push? just wanted to make sure that the dockerhub is only synced with the latest master

Reply:
Thanks a lot.  Yes, it is very important to make sure the docker image is built from master branch.  I added the condition to the docker push statement.